webapp/commonComponents/hooks/useFileDrop.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react'
+
+export default (elemRef, handler) => {
```

Comment:
we usually pass the handler function as first argument to hooks, or don't we?

Reply:
Move to first